index.js

```diff
@@ -137,13 +137,13 @@ export function plain(text, opts = {}, components = {}) {
   if (!text) return null;
   [text, opts] = setup(text, opts);
 
-  return htmlProcessor(opts)
-    .use(rehypeReact, {
-      createElement: React.createElement,
-      Fragment: React.Fragment,
-      components,
-    })
-    .processSync(text).contents;
+  const proc = htmlProcessor(opts).use(rehypeReact, {
+    createElement: React.createElement,
+    Fragment: React.Fragment,
+    components,
+  });
+
+  return proc.stringify(proc.runSync(proc.parse(text)));
```

Comment:
These _should_ be equivalent, but it was returning markdown rather than react?!

Reply:
Hmmm, this kinda stresses me out… Why was it returning the wrong thing? Will this have adverse effects on the ReadMe-side render?

Reply:
Only, if someone is grabbing the processor and calling `.processSync` directly? But, yea, maybe I should investigate more.

Reply:
True, just seems…I dunno…seems sketch that this would suddenly start returning something different???

Reply:
Ah, it was an intentional change, probably for `rehype-react`. If it's returning an object, it using `.result`, so it's not overloading the value of `.contents`: https://github.com/unifiedjs/unified/commit/c3ba172708e719baa963ee5826a7c81774a2784e